pytrip/vdx.py

```diff
@@ -1329,45 +1359,46 @@ def number_of_contours(self):
         """
         :returns: number of contours found in this Slice object.
         """
-        return len(self.contour)
+        return len(self.contours)
 
     def concat_contour(self):
         """ Concat all contours in this Slice object to a single contour.
         """
-        for i in range(len(self.contour) - 1, 0, -1):
-            self.contour[0].push(self.contour[i])
-            self.contour.pop(i)
-        self.contour[0].concat()
+        for i in range(len(self.contours) - 1, 0, -1):
```

Comment:
I know it is out of the scope of this PR, but what is the aim of this loop ?
Looks like an error-prone section.

Reply:
It is something @jakobtoftegaard wrote, I am not sure either what it does. Some sort of reverse loop, concating all contours starting at the last contour in the list or so.
Also the child/children concept is obscure, why it is needed.

That concat() method should probably be refactored.